p/pyyaml/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+	"maintainer": "pranith.rao",
+	"package_name": "pyyaml",
+	"github_url": "https://github.com/yaml/pyyaml",
+	"version": "6.0.2rc1",
+	"default_branch": "master",
+	"build_script": "pyyaml_ubi_9.3.sh",
+	"package_dir": "p/pyyaml",
+	"docker_build": false,
+	"validate_build_script": true,
+	"use_non_root_user": false,
+	"v1.*.*": {
```

Comment:
Version regex should match the versions being built. ex. `6.*.*` for your current scenario.